### DISCUSSION TITLE
맵 타일 뭐가 나음?

### DISCUSSION THREAD

**Discussion by boyism80** [KG-Crash/crash#39] [Q&A]
1. [Low Poly Hex Tiles Vol.1
](https://assetstore.unity.com/packages/3d/environments/fantasy/low-poly-hex-tiles-vol-1-120952)
2. [Low Poly Hexagon Tiles - Cartoon Pack](https://assetstore.unity.com/packages/3d/environments/low-poly-hexagon-tiles-cartoon-pack-148732)

**Comment by suhyeokkim** [comment]
2

**Comment by boyism80** [comment]
나는 1번

**Comment by Joogba** [comment]
1번으로 하겠습니다

**Reply by suhyeokkim** [reply]
힝
